### PR TITLE
Change vis_semantic_segmentation to take image

### PR DESCRIPTION
**Pull request by yuyu2172**
The interface is consistent with other visualizers.

### CODE REVIEW

File: tests/visualizations_tests/test_vis_semantic_segmentation.py [hunk 1]
```diff
@@ -16,17 +16,22 @@
...
     'all_label_names_in_legend': [False, True],
+    'no_img': [False, True],
 }))
 class TestVisSemanticSegmentation(unittest.TestCase):
 
     def setUp(self):
+        if hasattr(self, 'no_img'):
+            self.img = None
+        else:
+            self.img = np.random.randint(0, 255, size=(3, 32, 48))
```
Comment (Hakuyume) [inline tests/visualizations_tests/test_vis_semantic_segmentation.py]: All conditions have `self.no_img`.

Comment (yuyu2172) [reply tests/visualizations_tests/test_vis_semantic_segmentation.py]: sorry

Comment (Hakuyume) [reply tests/visualizations_tests/test_vis_semantic_segmentation.py]: `if self.no_img`

**Comment by Hakuyume** [comment]
LGTM

**Pull request merged**